package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-merge": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "npm run clean && npm run build && npm run test",
+    "prepare": "npm run clean && npm run build && npm run test",
```

Comment:
Currently, the build runs automatically after doing an `npm install`.  Are you saying that we don't want that?